minigu-test/Cargo.toml

```diff
@@ -5,9 +5,16 @@ name = "minigu-test"
 repository.workspace = true
 version.workspace = true
 
+[dependencies]
+glob = "0.3"
+libtest-mimic = "0.8.1"
+minigu = { workspace = true }
+sqllogictest = { workspace = true }
+
 [dev-dependencies]
 insta.workspace = true
 pastey.workspace = true
+tempfile = "3.0"
```

Comment:
`temp-file` is introduced in https://github.com/TuGraph-family/miniGU/pull/62. Maybe use this instead?